routes/api.php

```diff
@@ -30,13 +30,13 @@
     Route::get('/donation/{id}', [DonationController::class, 'show']);
     Route::post('/donations', [DonationController::class, 'store']);
     Route::put('/donations/{id}', [DonationController::class, 'update']);
+    Route::put('/update/{id}', [AuthController::class, 'update']);
```

Comment:
No es necesario poner la accion en la URI, podemos inferirlo con el metodo HTTP el cual espera la ruta (PUT...para editar recursos)

```suggestion
    Route::put('/{id}', [AuthController::class, 'update']);
```

Reply:
A mi me parece que es simple de leerlo y comprenderlo, porque al fin y al cabo PUT y UPDATE son conceptos que van a lo mismo pero no esta de mas aclaralo, para asi alguien que no este afin con el codigo pueda comprenderlo.

¿ Que decis ?

Reply:
Creo que es sobre explicar, lo cual no hace que nuestra interfaz luzca de calidad.